### ISSUE TITLE
DisplayCAL 3.9.14 for Windows no start sound

### ISSUE THREAD

**Issue by FQDianYan** [eoyilmaz/displaycal-py3#454]
**Describe the bug**
DisplayCAL 3.9.14 for Windows no start sound

**To Reproduce**
1. Start displaycal
2. See error

**Versions (please complete the following information):**
 - OS: Windows 11 24H2 10.0.26100.2033 x86_64
 - Python Version: 3.11
 - ArgyllCMS Version: 3.3.0
 - DisplayCAL Version: 3.9.14
**Additional context**
I'm already check ‘Enable start sound’.

**Comment by Headcrabed** [comment]
@eoyilmaz I am sure this issue still exists in 3.9.15 and the hint sound when calibrating is also can’t be played. Seems no log generated for this issue.

**Comment by eoyilmaz** [comment]
Yes I confirm this has not been solved in any OSes yet.

**Comment by St0RM53** [comment]
Yes i can confirm the issue is present on 3.9.15, you can see on the windows volume mixer displaycal doesn't appear, thus even if it generates a sound (startup/calibration etc.) it cannot play as it doesn't go through WDM.
